app/Controllers/feedController.php

```diff
@@ -647,6 +647,126 @@ public function deleteAction() {
 		}
 	}
 
+	/**
+	 * This action force to clear cache of a feed.
+	 *
+	 * Parameters are:
+	 *   - id (mandatory - no default): Feed ID
+	 *
+	 */
+	public function clearcacheAction() {
+		//Get Feed.
+		$id = Minz_Request::param('id');
+
+		$feedDAO = FreshRSS_Factory::createFeedDao();
+		$feed = $feedDAO->searchById($id);
+
+		if (!$feed) {
+			Minz_Request::bad(_t('feedback.sub.feed.not_found'), array());
+			return;
+		}
+
+		$feed->clearCache();
+
+		Minz_Request::good(_t('feedback.sub.feed.cache_cleared', $feed->name()), array(
+			'params' => array('get' => 'f_' . $feed->id())
+		));
+	}
+
+	/**
+	 * This action force to reload articles of a feed.
+	 *
+	 * Parameters are:
+	 *   - id (mandatory - no default): Feed ID
+	 *
+	 */
+	public function reloadAction() {
+		//Get Feed ID.
+		$feed_id = Minz_Request::param('id');
+
+		$feedDAO = FreshRSS_Factory::createFeedDao();
+		$entryDAO = FreshRSS_Factory::createEntryDao();
+
+		$feed = $feedDAO->searchById($feed_id);
+
+		if (!$feed) {
+			Minz_Request::bad(_t('feedback.sub.feed.not_found'), array());
+			return;
+		}
+
+		//Re-fetch articles as if the feed was new.
+		self::actualizeFeed($feed_id, null, false, null, true);
+
+		//Extract all feed entries from database, load complete content and store them back in database.
+		$entries = $entryDAO->listWhere('f', $feed_id, FreshRSS_Entry::STATE_ALL, 'DESC', 0);
+
+		$entryDAO->beginTransaction();
+
+		foreach ($entries as $entry) {
+			$entry->loadCompleteContent(true);
+			$entryDAO->updateEntry($entry->toArray());
+		}
+
+		$entryDAO->commit();
+
+		//Give feedback to user.
+		Minz_Request::good(_t('feedback.sub.feed.reloaded', $feed->name()), array(
+			'params' => array('get' => 'f_' . $feed->id())
+		));
+	}
+
+	/**
+	 * This action forge a preview of a content path.
```

Comment:
```suggestion
	 * This action creates a preview of a content path.
```
Forge has connotations like blacksmithing, or at least an awful lot of effort. ;-)

Reply:
Okay, it's an habit naming from my dev work.